tpm/constants.go

```diff
@@ -92,6 +154,47 @@ const (
 	rtTrans
 )
 
+// Locality type
+type Locality byte
+
+// Values of locality
+// Note: Localities are summable
+const (
+	locZero Locality = 1 << iota
+	locOne
+	locTwo
+	locThree
+	locFour
+)
+
+// LocaMap maps Locality values to strings for convenience
+var LocaMap = map[Locality]string{
```

Comment:
Consider making this private, so the caller has to use `.String()` to get the string representation.